.github/workflows/ci.yml

```diff
@@ -32,10 +32,12 @@ jobs:
       directory: 'terraform'
       terraform_version: '1.2'
 
+  # TODO(sethvargo): fix branch and remove golangci_url
   go_lint:
-    uses: 'abcxyz/pkg/.github/workflows/go-lint.yml@main' # ratchet:exclude
+    uses: 'abcxyz/pkg/.github/workflows/go-lint.yml@sethvargo/lint' # ratchet:exclude
     with:
       go_version: '1.19'
+      golangci_url: 'https://raw.githubusercontent.com/abcxyz/pkg/fddbebf/.golangci.yml'
```

Comment:
This is still an open AI because it depends on https://github.com/abcxyz/pkg/pull/50

Reply:
LGTM, other PR is approve if you wanna merge that and then fix this.